functions/go/set-namespace/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM golang:1.14
+FROM golang:1.13-stretch
 ENV CGO_ENABLED=0
 WORKDIR /go/src/
 COPY . .
-RUN go build -v -o /usr/local/bin/set-namespace ./
-
+RUN go build -v -o /usr/local/bin/function ./
 FROM alpine:latest
```

Comment:
We should standardize this base image across Go and TS functions.

Reply:
`alpine:latest` may not be great
IMO we should use `alpine:3.12` for go functions and `node:lts-alpine3.12` for TS functions.
It's probably safe to use the latest patch releases. `alpine:3.12` points to the latest 3.12.* versions.

Reply:
@mengqiy It will be good to create an issue to discuss it.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/1247

Reply:
I have added a section in my doc and you guys should get an email notification for that. PTAL.